### ISSUE TITLE
Displaying nickname instead of full name

### ISSUE THREAD

**Issue by Brume-AL** [atari-legend/atari-legend#89]
In the donator part, I really prefer to display nickname instead of real name. I remember some guys didn't want to see their real name on Atari Legend (Nicolas B, Ruthless...). That's why I always used the aka on Stonish.

Or maybe should I remove some of real names? Since these information weren't displayed on the site, I just added them for myself (like a reminder).

![Atari Legend Menu](https://user-images.githubusercontent.com/12120244/108198266-b59db780-711b-11eb-850c-f0c419969511.jpg)

**Comment by nguillaumin** [comment]
Fair point. One challenge is that in the AL DB we don't really have the concept of a "main name" (or real name) and nicknames. It just relies on the actual association to an individual is made on the real name individual rather than a nick (which is what I tried to do when importing the data from Stonish).

Anyway I changed it to pick the first nick name we have for the individual, which should work in practice.
